hnn_core/viz.py

```diff
@@ -402,9 +424,28 @@ def plot_spikes_raster(cell_response, ax=None, show=True):
     """
 
     import matplotlib.pyplot as plt
-    spike_times = np.array(sum(cell_response._spike_times, []))
-    spike_types = np.array(sum(cell_response._spike_types, []))
-    spike_gids = np.array(sum(cell_response._spike_gids, []))
+    n_trials = len(cell_response.spike_times)
+    if trial_idx is None:
+        trial_idx = list(range(n_trials))
+
+    _validate_type(trial_idx, (int, list), 'trial_idx', 'int, list of int')
+
+    if isinstance(trial_idx, int):
+        trial_idx = [trial_idx]
+
+    # Extract desired trials
+    if cell_response._spike_times[0]:
+        spike_times = np.concatenate(
+            np.array(cell_response._spike_times)[trial_idx])
+        spike_types = np.concatenate(
+            np.array(cell_response._spike_types)[trial_idx])
+        spike_gids = np.concatenate(
+            np.array(cell_response._spike_gids)[trial_idx])
```

Comment:
what is more desirable if multiple trials are included? plot each trial as separate subplots or plot them all in one plot?